src/clj_kondo/impl/linters.clj

```diff
@@ -475,19 +475,47 @@
            (node->line filename node
                        finding-type msg)))))))
 
-(defn lint-unused-bindings!
+(defn lint-bindings!
   [ctx]
   (doseq [ns (namespace/list-namespaces ctx)]
     (let [ns-config (:config ns)
           ctx (if ns-config
                 (assoc ctx :config ns-config)
                 ctx)
           ctx (assoc ctx :lang (:lang ns))]
+      (when-not (identical? :off (-> ctx :config :linters :used-underscored-binding :level))
+        (doseq [binding (set (->> (:used-bindings ns)
+                                  (filter (comp not :clj-kondo.impl/generated))
+                                  (filter #(str/starts-with? (str (:name %)) "_"))))
+                :when (not= "_" (str (:name binding)))]
```

Comment:
what is the use case for this line?

Reply:
you're right, makes no sense to ignore the `(let [_ 1] _)` case... i'll update.